packages/aragon-cli/src/commands/dao_cmds/new.js

```diff
@@ -137,7 +137,9 @@ exports.task = async ({
           // Backward compatibility with old event name
           const deployEventValue =
             events[deployEvent] ||
-            events[exports.OLD_BARE_TEMPLATE_DEPLOY_EVENT]
+            events[exports.OLD_BARE_TEMPLATE_DEPLOY_EVENT] ||
+            // Some templates use DeployDAO instead of DeployDao
+            events['DeployDAO']
```

Comment:
🤦‍♂ are these the old kits?

Reply:
The problem was in the boilerplate: https://github.com/aragon/aragon-react-boilerplate/blob/558074e1863bbcb44cb0d2c6470866bbd94ca40e/contracts/Template.sol#L30
But it seems to be fixed now.